packages/algjulia-service/src/decapodes-service/theory.jl

```diff
@@ -0,0 +1,138 @@
+""" Helper function to convert CatColab values (Obs) in Decapodes """
+function to_theory(theory::ThDecapode, type::ObType, name::String)
+    @match lowercase(name) begin
+        "0-form" => :Form0
+        "1-form" => :Form1
+        "2-form" => :Form2
+        "primal 0-form" => :Form0
+        "primal 1-form" => :Form1
+        "primal 2-form" => :Form2
+        "dual 0-form" => :DualForm0
+        "dual 1-form" => :DualForm1
+        "dual 2-form" => :DualForm2
+        x => throw(ImplError(x))
+    end
+end
+
+""" Helper function to convert CatColab values (Homs) in Decapodes """
+function to_theory(theory::ThDecapode, type::HomType, name::String)
+    @match replace(name," " => "") begin
+        "∂t" || "∂ₜ" => :∂ₜ
+        # "∂ₜ" => :∂ₜ
+        "Δ" => :Δ
+        "Δ⁻¹" => :Δ⁻¹
+        "d*" || "d̃₁" => :dual_d₁
+        # \star on LHS
+        "⋆" => :⋆₁
+        "⋆⁻¹" || "⋆₀⁻¹" => :⋆₀⁻¹
+         # => :⋆₀⁻¹
+        # \bigstar on LHS
+        "★" || "★⁻¹" => :⋆₁
+         # => :⋆₀⁻¹
+        "diffusivity" => :diffusivity
```

Comment:
Are we still hard-coding the list of constants? That should be fixed, though I won't insist that it happen in this PR if it's feels like too much work.

Reply:
I'll remove this and test, but to keep pushes more frequent I've pushed the rest of the feedback

Reply:
Marking as resolved since this is now captured by #284.